tests/gordo_components/builder/test_builder.py

```diff
@@ -262,6 +294,174 @@ def test_get_metadata_helper(model: BaseEstimator, expect_empty_dict: bool):
+                        transformer: sklearn.preprocessing.data.MinMaxScaler
+                        regressor:
+                            sklearn.pipeline.Pipeline:
+                                steps:
+                                - sklearn.preprocessing.data.MinMaxScaler
+                                - gordo_components.model.models.KerasAutoEncoder:
+                                    kind: feedforward_hourglass
+                                    batch_size: 3
+                                    compression_factor: 0.5
+                                    encoding_layers: 1
+                                    func: tanh
+                                    out_func: linear
+                                    epochs: 1
+            """
+
+    model_config = yaml.load(raw_model_config, Loader=yaml.FullLoader)
+    model, metadata = build_model(
+        name="model-name",
+        model_config=model_config,
+        data_config=data_config,
+        metadata={},
+    )
+
+    assert (
+        metadata["model"]["cross-validation"]["scores"][
+            "explained-variance-score-Tag-1"
+        ]["fold-mean"]
+        + metadata["model"]["cross-validation"]["scores"][
+            "explained-variance-score-Tag-2"
+        ]["fold-mean"]
+    ) / 2 == pytest.approx(
+        metadata["model"]["cross-validation"]["scores"][
+            "explained-variance-score-tag-mean"
+        ]["fold-mean"]
+    )
+
+    assert (
+        metadata["model"]["cross-validation"]["scores"]["r2-score-Tag-1"]["fold-mean"]
+        + metadata["model"]["cross-validation"]["scores"]["r2-score-Tag-2"]["fold-mean"]
+    ) / 2 == pytest.approx(
+        metadata["model"]["cross-validation"]["scores"]["r2-score-tag-mean"][
+            "fold-mean"
+        ]
+    )
+
+    assert (
+        metadata["model"]["cross-validation"]["scores"]["r2-score-Tag-1"]["fold-mean"]
+        + metadata["model"]["cross-validation"]["scores"]["r2-score-Tag-2"]["fold-mean"]
+    ) / 2 == pytest.approx(
+        metadata["model"]["cross-validation"]["scores"]["r2-score-tag-mean"][
+            "fold-mean"
+        ]
+    )
+
+    assert (
+        metadata["model"]["cross-validation"]["scores"]["mean-squared-error-Tag-1"][
+            "fold-mean"
+        ]
+        + metadata["model"]["cross-validation"]["scores"]["mean-squared-error-Tag-2"][
+            "fold-mean"
+        ]
+    ) / 2 == pytest.approx(
+        metadata["model"]["cross-validation"]["scores"]["mean-squared-error-tag-mean"][
+            "fold-mean"
+        ]
+    )
+
+    assert (
+        metadata["model"]["cross-validation"]["scores"]["mean-absolute-error-Tag-1"][
+            "fold-mean"
+        ]
+        + metadata["model"]["cross-validation"]["scores"]["mean-absolute-error-Tag-2"][
+            "fold-mean"
+        ]
+    ) / 2 == pytest.approx(
+        metadata["model"]["cross-validation"]["scores"]["mean-absolute-error-tag-mean"][
```

Comment:
again for readability, creating a variable `scores = metadata["model"]["cross-validation"]["scores"]` would save on readability in this test.